.goreleaser.yml

```diff
@@ -0,0 +1,28 @@
+# Configuration file for GoReleaser - https://goreleaser.com/build/
+builds:
+- env:
+    - CGO_ENABLED=0
+  goos:
+    - darwin
+    - linux
+    - windows
+  goarch:
+    - amd64
+
+before:
+  hooks:
+    - dep ensure
+
+archive:
```

Comment:
Will a github release only contain the binaries?

Reply:
As I understood it, notes of sha265 for the files as well as the commit history will be produced as notes when releasing.

Reply:
Regarding the readme bump, I think it would be nice with a "correct" URL to make it easier to download. @FredrikFolkesson is there any way you can hook into a release and add this?

Reply:
The github release will be 3 zipped files (which only includes the binaries)

SHA:s and changelog should as I understand it be added to the release itself.

@carlioth 
There does not seem to be a way to link to a artifact from "latest" release :( Since we are only pushing tags I dont think we can do any changes.

I think changing the wording to <version> or something should be good enough.